circle.yml

```diff
@@ -7,7 +7,7 @@ machine:
     ADMINS: Olivier Le Floch <olefloch@work4labs.com>
     SECRET_KEY: EamJrjviiW13ue8PEamJrjviiW13ue8P
   python:
-    version: 2.7.9
+    version: 2.7.11
```

Comment:
https://developer.rackspace.com/blog/the-not-so-sorry-state-of-ssl-in-python/ SNI & co
Of course none of that is likely to have an impact on the shortener, but I'm raising this issue in the context of our other projects.